db_sql.go

```diff
@@ -893,6 +919,8 @@ func createMigrations() []migration.Migrator {
 
 		CREATE TABLE authpwdarchive (id BIGSERIAL PRIMARY KEY, userid BIGINT NOT NULL, password VARCHAR NOT NULL, created TIMESTAMP DEFAULT NOW());
 		`,
+		// 11. Account lock
+		`ALTER TABLE authuserpwd ADD COLUMN accountlocked BOOLEAN;`,
```

Comment:
What's the policy with NULL? Does NULL mean locked or not locked? Do new entries in the table get it set to false or NULL?